go/logic/applier.go

```diff
@@ -52,61 +53,82 @@ func newDmlBuildResultError(err error) *dmlBuildResult {
 // Applier is the one to actually write row data and apply binlog events onto the ghost table.
 // It is where the ghost & changelog tables get created. It is where the cut-over phase happens.
 type Applier struct {
-	connectionConfig  *mysql.ConnectionConfig
-	db                *gosql.DB
-	singletonDB       *gosql.DB
-	migrationContext  *base.MigrationContext
-	finishedMigrating int64
+	srcConnectionConfig *mysql.ConnectionConfig
+	dstConnectionConfig *mysql.ConnectionConfig
+	srcDB               *gosql.DB
+	dstDB               *gosql.DB
+	singletonDB         *gosql.DB
+	migrationContext    *base.MigrationContext
+	finishedMigrating   int64
 }
 
 func NewApplier(migrationContext *base.MigrationContext) *Applier {
 	return &Applier{
-		connectionConfig:  migrationContext.ApplierConnectionConfig,
-		migrationContext:  migrationContext,
-		finishedMigrating: 0,
+		srcConnectionConfig: migrationContext.SrcMasterConnectionConfig,
+		dstConnectionConfig: migrationContext.DstMasterConnectionConfig,
+		migrationContext:    migrationContext,
+		finishedMigrating:   0,
 	}
 }
 
 func (this *Applier) InitDBConnections() (err error) {
 
-	applierUri := this.connectionConfig.GetDBUri(this.migrationContext.DatabaseName)
-	if this.db, _, err = mysql.GetDB(this.migrationContext.Uuid, applierUri); err != nil {
+	srcApplierURI := this.srcConnectionConfig.GetDBUri(this.migrationContext.SrcDatabaseName)
```

Comment:
srcApplierURI name should not contain "Applier"，confirm to Inspector  https://github.com/github/gh-ost/pull/565/files#diff-5aa50dde7ca1baf7a819eecf740ea220R51